Include/internal/pycore_interp.h

```diff
@@ -295,8 +295,11 @@ _PyInterpreterState_SetFinalizing(PyInterpreterState *interp, PyThreadState *tst
 }
 
 
+extern int64_t _PyInterpreterState_ObjectToID(PyObject *);
+
 // Export for the _xxinterpchannels module.
 PyAPI_FUNC(PyInterpreterState *) _PyInterpreterState_LookUpID(int64_t);
+PyAPI_FUNC(PyInterpreterState *) PyInterpreterState_LookUpIDObject(PyObject *);
```

Comment:
Shouldn't this have a leading `_` as well, like the others below?

Reply:
Yeah, I'll fix that.

Reply:
done